### ISSUE TITLE
Emit morse code using phone flash light

### ISSUE THREAD

**Issue by Siltaar** [sussman/androidomatic-keyer#31]
It would be useful to be able to emit (for instance SOS) morse code via the camera flash light on the back of most Android phones instead of emitting morse code via sounds.

**Comment by filobus** [comment]
It would be interesting also to have these flashlights (or sounds) be decoded if i observe them with my camera (+phone and this app) from distance! (Not an easy task to implement!)

**Comment by goloszi** [comment]
Yes, I would like to have that feature too

**Comment by phishbowler** [comment]
Using the flashlight to produce morse code would make the app very powerful, especially as a rescue tool.

**Comment by ghost** [comment]
Yeah, sound can only travel so far, but light can be seen for miles on a dark night. Any updates on this?

**Comment by iandeare** [comment]
There are strobe apps for 'phones. Not Morse (which is no longer officially recognised as a language; but there are still users) the one I have is an unstable bets version, and power hungry

**Comment by iandeare** [comment]
Strobily

**Comment by Innovrmyhead** [comment]
I know a previous request was for the pH to emit a sound along with the flash light which is an excellent idea that makes sound should be something here wrenching and just a sound that would be totally distinct SQL in a most environments definitely screen that it is man making this noise through a device,!!  But what would really be awesome also what do you need for a way to read the flashes or the sounds from far and interpret that back into layman's terms so if you seeing a flashing light or heard the morse code in the wilderness or seen it so far most people don't know more stuff so they wouldn't know what the people were trying to say! I don't know if you can that would be awesome if there was a way to read Morse code also receive it out guessing you have to be for the camera that would be an awesome feature!!! Your App would truly be complete!
